config.go

```diff
@@ -4,67 +4,67 @@ package specs
 // independent configuration.
 type Spec struct {
 	// Version is the version of the specification that is supported.
-	Version string `json:"version"`
+	Version string `json:"version" mandatory:"required"`
```

Comment:
Since the similar 'tag' will be generated automaticly by the protobuf golang compiler, and proto.Marshal makes thing easier. I'll write another implementation based on that.